web/yarn.lock

```diff
@@ -0,0 +1,2089 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Hi 
I don't think we need to keep this .lock file and fix the version because package dependencies are written to be compatible. Do we need this lock file?

Reply:
Hi, I would keep the yarn.lock file for. contributors: [reason](https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for)

Reply:
Thanks all, I read this official [document](https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/).

> Without lockfiles it gets even more complicated: In applications or libraries, if there is no lockfile, you will have to check the dependencies every time you install or re-install them and make sure that everything still works.

Let's follow official way so user can respect proper dependencies :)

Reply:
Oh! Thank you for information !!

Reply:
Yes! It's absolutely library! so the library user doesn't matter, but having lock file would help the contributor :P